persistence/actor.go

```diff
@@ -80,6 +80,23 @@ func (p *PostgresContext) GetAllValidators(height int64) (vals []*coreTypes.Acto
 	return
 }
 
+// GetValidatorSet returns the validator set for a given height
+func (p *PostgresContext) GetValidatorSet(height int64) (*coreTypes.ValidatorSet, error) {
+	validators, err := p.GetAllValidators(height) // sorted by address+height desc
```

Comment:
We're only getting the validators at that height, right?

```suggestion
	validators, err := p.GetAllValidators(height) // sorted by address desc
```

Reply:
So the `GetAllValidators` function uses this in the SQL query `height <= %d` and orders by address and height desc. This means we dont just get the validators staked at the height provided but those who are able to vote at the height provided+1 (aka, the entire validator set for the block after the height provided)